### DISCUSSION TITLE
Upgrade to openmage from 1.9.2.4

### DISCUSSION THREAD

**Discussion by bylli79** [OpenMage/magento-lts#2922] [Questions & Answers] [answered]
Hi,
I have a version of magento 1.9.2.4.
I would like to switch to openmage.
Can I update magento from 1.9.2.4 to 1.9.4.5 or do I have to go through the intermediate versions?

Thank you

**Comment by fballiano** [comment]
it's not necessary to do the intermediate upgrades

**Comment by addison74** [comment, marked as answer]
Last year I made an upgrade from Magento 1.9.2.4 to OM 20. Here are the steps I took:

- I created a backup of the file system
- I created a full dump of the database
- I took them to a test environment similar to the new server where I was going to use them.
- I analyzed all differences between files/directories from the backup and Magento 1.9.2.4 using a software (Araxis Merge). There are situations when certain developers did not respect the rules and even modified the core files.
- When I decided what I had to keep and what I had to delete, I moved on removing everything related to Magento 1.9.2.4 (files and folders)
- I copied the OpenMage archive directly into the file system and started testing. The website loaded immediately and I activated the log files in the Backend (I will find them in /var/log/). Certain extensions had issues with the PHP version I used (7.4) and I had to do some changes this step consumed a lot of time, I am not sure they are all perfect but the shopping cart works.

Out of curiosity, this Magento version 1.9.2.4, quite stable in my opinion, no longer allows you to upload images because it used Flash Uploader. No browser currently supports Flash Player.
